woocommerce/woocommerce
# Update WooPayments task copy

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes #43296 

Update WooPayments task copy according to paJDYF-bGl-p2

### How to test the changes in this Pull Request:

>[!Note]
> Snapshot builds seem to interfere with [this condition](https://github.com/woocommerce/woocommerce/blob/bd1351e265c865de9f632bfb978ebc01c8f9823a/plugins/woocommerce/src/Admin/Features/PaymentGatewaySuggestions/DefaultPaymentGateways.php#L847-L864), causing it to load generic payments task instead of WooPayments one. You can build WC locally or rename the plugin folder to `woocommerce` after installing the snapshot.

- Create a new site with WC from this PR branch.
- Skip the profiler, and pick a WooPayments supported country (e.g. US).
- Install WooPayments, but don't onboard.
- Payments (1) menu should be visible (action incentive).
- Go to WC Home.
- `Set up WooPayments` task should match the attached screenshot.

### Screenshot

| Before | After |
|--------|--------|
|<img width="672" alt="Screenshot 2024-01-08 at 14 33 09" src="https://github.com/woocommerce/woocommerce/assets/7670276/d678a188-b408-420c-87d6-1fe852a91dff">|<img width="670" alt="Screenshot 2024-01-08 at 14 31 40" src="https://github.com/woocommerce/woocommerce/assets/7670276/b87aa725-1a0e-4a16-81a7-3a3e1aa8e712">|
